polyfill.js

```diff
@@ -0,0 +1,11 @@
+if (typeof window.Map === 'undefined') {
+	window.Map = require('can-cid/map/map');
+}
+
+if (typeof window.Set === 'undefined') {
+	window.Set = require('can-cid/set/set');
+}
+
+window.requestAnimationFrame = function(callback) {
```

Comment:
Why no check for RAF before assignment? Looks like you'd be destroying RAF for everyone as it is.

Reply:
Because i like to break things